package.json

```diff
@@ -1,39 +1,42 @@
 {
-  "name": "sui-monorepo",
-  "private": true,
-  "license": "Apache-2.0",
-  "scripts": {
-    "turbo": "turbo",
-    "core": "pnpm --filter ./apps/core",
-    "icons": "pnpm --filter ./apps/icons",
-    "explorer": "pnpm --filter ./apps/explorer",
-    "wallet": "pnpm --filter ./apps/wallet",
-    "wallet-adapter": "pnpm --filter ./sdk/wallet-adapter",
-    "wallet-kit-site": "pnpm --filter wallet-kit-site",
-    "sdk": "pnpm --filter ./sdk/typescript",
-    "bcs": "pnpm --filter ./sdk/bcs",
-    "kiosk": "pnpm --filter ./sdk/kiosk",
-    "suins": "pnpm --filter ./sdk/suins-toolkit",
-    "deepbook": "pnpm --filter ./sdk/deepbook",
-    "changeset-version": "pnpm changeset version && pnpm sdk codegen:version"
-  },
-  "pnpm": {
-    "overrides": {
-      "node-notifier": "10.0.0",
-      "async": "3.2.2",
-      "nth-check": "2.0.1",
-      "yaml@<2.2.2": ">=2.2.2"
+    "name": "sui-monorepo",
+    "private": true,
+    "license": "Apache-2.0",
+    "scripts": {
+        "turbo": "turbo",
+        "core": "pnpm --filter ./apps/core",
+        "icons": "pnpm --filter ./apps/icons",
+        "explorer": "pnpm --filter ./apps/explorer",
+        "wallet": "pnpm --filter ./apps/wallet",
+        "wallet-adapter": "pnpm --filter ./sdk/wallet-adapter",
+        "wallet-kit-site": "pnpm --filter wallet-kit-site",
+        "sdk": "pnpm --filter ./sdk/typescript",
+        "bcs": "pnpm --filter ./sdk/bcs",
+        "kiosk": "pnpm --filter ./sdk/kiosk",
+        "suins": "pnpm --filter ./sdk/suins-toolkit",
+        "deepbook": "pnpm --filter ./sdk/deepbook",
+        "changeset-version": "pnpm changeset version && pnpm sdk codegen:version"
+    },
+    "prettier": "./.config/prettier.config.js",
```

Comment:
Does this directory already exist or was it just created for this? Curious why this vs a file in the root

Reply:
this exists, my rough understanding is that `.config` directories are party intended to avoid the issue of cluttering the root with configs for a million different tools.   I went back and forth on this a bit, but I think maybe just moving it out to the root is fine here, since the `.prettierignore` needs to be in the root anyway

Reply:
I don't have a strong preference here, so don't stress on changing this.